application/templates/base.html

```diff
@@ -56,6 +56,9 @@
         <script src="/static/vendor/autotrack/autotrack.js"></script>
     {% endif %}
 
+    {% if not static_mode %}
```

Comment:
@TheDoubleK I considered this when looking at the `base.html` template, but discarded it on the grounds that it feels like it's good to move towards removing differences between the pages in "static" mode vs the pages in "Flask app" mode.

For example, removing the CMS css from the homepage and topic pages only on static site mode would mean that if we accidentally added some styles needed for the public homepage/topic page to the CMS css, we'd only notice once the site had been deployed, not when browsing the homepage on the review app.

An alternative approach might be to just merge the CMS and application CSS files together, given than there's now not _that_ much extra CSS in the CMS stylesheet anyway. This would simplify things, reduce errors, and the only downside would be a marginal few extra unnecessary kb of CSS for public users to download.